src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem/TodoItem';
+
+interface Props {
+  todos: Todo[],
+}
+
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <section className="todoapp__main" data-cy="TodoList">
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings